travis/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+cd $(dirname $0)/..
+sudo prove -rv .
```

Comment:
This will create the temporary files in the source directory, which isn't what we want if the source directory is a network filesystem.  Should we use /tmp instead?